e2e/tenv.ts

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable import/no-nodejs-modules */
```

Comment:
Why is it needed ?

Reply:
Because eslint is erroring. Don't want to fight with it, it's just for this file that we want Node modules.